src/test/scala/intent/helpers/TestSuiteRunnerTester.scala

```diff
@@ -9,21 +9,21 @@ import intent.runner.{TestSuiteRunner, TestSuiteError, TestSuiteResult}
 */
 trait TestSuiteRunnerTester(given ec: ExecutionContext) extends Subscriber[TestCaseResult] with
 
-  def suiteClassName: String
+  def suiteClassName: String | Null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.